source/style.css

```
@@ -307,6 +306,7 @@ body.nerd-info :is(#nerd-info, #nerd-log) {
 	background-color: rgb(119 119 119);
 	overflow-x: auto;
 	overflow-y: hidden;
+	margin: 13px 0;
```

```suggestion
	margin: 1vh 0;
```
It is not a good idea to mix pixels with percentage-based values